ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java

```diff
@@ -298,13 +297,9 @@ public void getOptimizeInfo(Context ctx) {
     Preconditions.checkState(tableService.tableExist(new com.netease.arctic.ams.api.TableIdentifier(catalog, db,
         table)), "no such table");
 
-    List<OptimizedRecord> all = tableDescriptor.getOptimizeInfo(catalog, db, table);
-    List<OptimizedRecord> result =
-        all.stream().sorted(Comparator.comparingLong(OptimizedRecord::getCommitTime).reversed())
-            .skip(offset)
-            .limit(limit)
-            .collect(Collectors.toList());
-    ctx.json(OkResponse.of(PageResult.of(result, all.size())));
+    PageResult<TableOptimizingProcess> result =
+        tableDescriptor.getOptimizingProcesses(catalog, db, table, offset, limit);
+    ctx.json(OkResponse.of(result));
```

Comment:
The processing logic here is inconsistent with the processing approach at line 238 in ServerTableDescriptor.java. Do we need to consider the limit restriction in the Derby database or not?

Reply:
Unless we remove support for Derby now, I think we need to consider the limit restriction in the Derby database.

Reply:
I have changed it.